core/src/main/java/com/redhat/lightblue/client/response/DefaultLightblueDataResponse.java

```diff
@@ -157,4 +77,12 @@ public JsonNode getProcessed() {
         }
     }
 
+    protected int parseInt(String fieldName) {
+        JsonNode field = getJson().findValue(fieldName);
+        if (field == null || field.isNull()) {
```

Comment:
findValue(x) searches x recursively in the node, or its children. Is this really what you want here? I think you should use get(x).